src/EnergyPlus/InputProcessing/InputProcessor.hh

```diff
@@ -320,7 +320,7 @@ private:
     std::unique_ptr<IdfParser> idf_parser;
     std::unique_ptr<Validation> validation;
     std::unique_ptr<DataStorage> data;
-    json schema;
```

Comment:
So before, `schema` was just a member variable on the input processor, and now it is changed to a function.  OK.